dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/MergeUser/ChooseTrn.cshtml

```diff
@@ -0,0 +1,26 @@
+@page "/admin/users/{userId}/merge/{UserIdToMerge}/choose-trn"
+@model TeacherIdentity.AuthServer.Pages.Admin.MergeUser.ChooseTrn
+@{
+    ViewBag.Title = Html.DisplayNameFor(m => m.Trn);
+}
+
+@section BeforeContent {
+    <govuk-back-link asp-page="Confirm" asp-route-userId="@Model.UserId" asp-route-userIdToMerge="@Model.UserIdToMerge"/>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+        <form asp-page="ChooseTrn" method="post">
+            <govuk-radios asp-for="Trn">
+                <govuk-radios-fieldset>
+                    <govuk-radios-fieldset-legend class="govuk-fieldset__legend--xl"/>
+
+                    <govuk-radios-item value="@(Model.User!.Trn ?? "None")">@(Model.User!.Trn ?? "None")</govuk-radios-item>
```

Comment:
N/A is used on User page, but didn't seem to make sense here. Should we allow None as an option?

Reply:
Agree, 'N/A' seems weird here - 'None' makes sense to me. Would be good to only have a single option here if neither user has a TRN, though (instead of getting two 'None' options).

Reply:
Ideally the user wouldn't see this page if both are None. You've seen my comment below! Is this ok as is?